test/ibmq/test_ibmq_integration.py

```diff
@@ -143,3 +148,84 @@ def test_execute_two_remote(self, provider):
         job = execute([qc, qc_extra], backend, seed_transpiler=self.seed)
         results = job.result()
         self.assertIsInstance(results, Result)
+
+    def test_ensure_backend_jobs_signature(self):
+        """Test `IBMQBackend.jobs` signature is similar to `IBMQBackendService.jobs`
+
+        The signature of `IBMQBackend.jobs` is similar to the signature of
+        `IBMQBackendService.jobs` if its parameter list is a subset of the
+        parameter list of `IBMQBackendService.jobs`.
+        """
+        backend_cls = IBMQBackend
+        backend_service_cls = IBMQBackendService
+
+        # Retrieve parameter lists for both classes.
+        backend_jobs_args = getattr(
+            getfullargspec(backend_cls.jobs), 'args', [])
+        provider_backends_jobs_args = getattr(
+            getfullargspec(backend_service_cls.jobs), 'args', [])
+
+        # Ensure parameter lists not empty
+        self.assertTrue(backend_jobs_args)
+        self.assertTrue(provider_backends_jobs_args)
+
+        if not all(arg in provider_backends_jobs_args for arg in backend_jobs_args):
```

Comment:
This will not catch the case wherein a developer adds a parameter to `IBMQBackendService` but forgot to add it to `IBMQBackend`. Really the signatures should be the same other than the `backend` keyword.